src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -2,18 +2,48 @@
 
 <Project>
  
-  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
-  
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+  <UsingTask TaskName="GetLastStablePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+
   <Target Name="RunPackageValidation" AfterTargets="Pack">
+    
+    <GetLastStablePackage PackageId="$(PackageId)"
```

Comment:
Consider avoiding the repetitive task invocation and with that numerous IO calls (depending on the already cached versions of a package) by either:
- Storing the path to the previous package in the intermediate directory of the app and reading from it.
- Reading the project.assets.json/deps.json to get the path to the package

@ericstj opinions?

Reply:
Yeah, we should emit a props file during restore, otherwise we have a consistency issue. Bummer that PackageDownload doesn’t honor GeneratePathProperty.